hw/ip/spi_device/doc/_index.md

```diff
@@ -410,6 +410,13 @@ Except BUSY bit, other bits are controlled by SW.
 BUSY bit is set by HW when it receives any commands that are uploaded to the FIFOs.
 SW may clear BUSY bit when it completes the received commands (e.g Erase/ Program).
 
+If BUSY is set, SPI_DEVICE IP blocks the passhthrough interface in Passthrough mode.
+The blocking of the interface occurs in SPI transaction idle state (CSb == 1).
+However, due to the instrinsic delay of the CDC, CSb transition is delayed by 2 SYS_CLK cycles.
+It may introduce a corner case when the host system starts sending a SPI transaction while SW may clear the BUSY signal.
+In that case, the blocking and unblocking of the passthrough may happen while SPI is active.
+The HW behavior in this scenario is not determined.
```

Comment:
This scenario can be avoided by observing the minimum CSb high time. (long enough to cover the max delay through the CDC)

And that's only true if changes are limited to the _rising edge_ of CSb (more specific trigger than CSb == 1). It looks like this is what you have implemented already in `spid_status`, for what it's worth. :)

Reply:
yes